drivers/src/pci/root.rs

```diff
@@ -55,6 +65,46 @@ impl PciBarType {
             _ => None,
         }
     }
+
+    /// Reads a PCI BAR resource type from the first cell in a PCI address range.
```

Comment:
Nit: Non-public function should probably use regular comment instead of doc-style.

Reply:
Done.